### PR TITLE
Remove references to DeploymentConfig. Should double check.

### PR DESCRIPTION
**Pull request by lvarin**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/DeploymentConfig-nomore/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/DeploymentConfig-nomore/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/rahti2/concepts.md [hunk 1]
```diff
@@ -302,20 +329,19 @@ OpenShift includes all Kubernetes objects, plus some extensions:
...
 * **Route** objects connects a **Service** with the internet using _HTTP_.
 
 ### DeploymentConfig
 
 !!! Warning
 
-    Starting OKD 4.14, DeploymentConfig API is being deprecated. Only security-related and critical issues will be fixed in the future.  
-    More information [here](https://access.redhat.com/articles/7041372)
+    Starting OKD 4.14, DeploymentConfig API is being deprecated. Only security-related and critical issues will be fixed in the future.
+    More information [here](https://access.redhat.com/articles/7041372) and the [replacement guide for DeploymentConfig](https://developers.redhat.com/learning/learn:openshift:replace-deprecated-deploymentconfigs-deployments/resource/resources:convert-deploymentconfig-deployment).
```
Comment (vartikashah) [inline docs/cloud/rahti2/concepts.md:340]: Do we need to update apiVersion in 
deploymentconfig.yaml to "apps.openshift.io/v1" and in 
buildconfig.yaml to "build.openshift.io/v1"

Comment (lvarin) [reply docs/cloud/rahti2/concepts.md:340]: fixed, thanks

File: docs/cloud/tutorials/helm.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Now we can create the `YAML` files that contain the different parts of the appli
 
 	```sh
 	$ oc get dc,deployment,pvc,secret,configmaps,service,route -o name
```
Comment (vartikashah) [inline docs/cloud/tutorials/helm.md:77]: Can remove dc ?

File: docs/cloud/tutorials/helm.md [hunk 3]
```diff
@@ -548,27 +502,26 @@ appVersion: "1.16.0"
 * `templates/deployment.yaml`:
```
Comment (vartikashah) [inline docs/cloud/tutorials/helm.md:502]: To upgrade the helm chart. Command working for me is:

$ helm upgrade test-name example

File: docs/cloud/tutorials/helm.md [hunk 4]
```diff
@@ -91,9 +91,9 @@ Now we can create the `YAML` files that contain the different parts of the appli
...
+1. From the list above, we are only interested in `deployment.apps`, `persistentvolumeclaim/html`, `service/nginx` and `route.route.openshift.io/nginx`. The rest are auto-generated like the `secret/` tokens or created by other objects like the `service/glusterfs-dynamic-ed156002-8a7e-11ed-b60d-fa163e0d8841` object was created as a result of the creation of the `persistentvolumeclaim/` (PVC) creation.
 
-We will write templates one by one, starting with the Volume. There are two simple approaches to accomplish this task, "get and and clean" or "recreate from template". We will first try the "get and clean" method.
+We will write templates one by one, starting with the Volume. There are two simple approaches to accomplish this task, "get and clean" or "recreate from template". We will first try the "get and clean" method.
 
 ### Get and clean
```
[the remark this answers is not in the document]
Comment (lvarin) [reply docs/cloud/tutorials/helm.md:99]: ?

I am confused

File: docs/cloud/tutorials/helm.md (hunk 3 above)
Comment (lvarin) [reply docs/cloud/tutorials/helm.md:502]: where is this mentioned? Could make the comment under the line that you are talking about?

File: docs/cloud/tutorials/helm.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lvarin) [reply docs/cloud/tutorials/helm.md:99]: Could make the comment under the line that you are talking about?

[the remark this answers is not in the document]
Comment (vartikashah) [reply docs/cloud/tutorials/helm.md:99]: Line 159.

File: docs/cloud/tutorials/helm.md (hunk 3 above)
Comment (vartikashah) [reply docs/cloud/tutorials/helm.md:502]: Line 365

**Review by vartikashah — APPROVED** [review]
LGTM

File: docs/cloud/tutorials/helm.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lvarin) [reply docs/cloud/tutorials/helm.md:99]: To b done in a separate PR

**Pull request merged**